binr/ragg2/Makefile

```diff
@@ -3,6 +3,7 @@ BINDEPS=r_egg r_bin r_syscall r_asm
 BINDEPS+=r_io r_socket
 BINDEPS+=r_lang r_parse r_anal r_reg r_flag
 BINDEPS+=r_magic r_cons r_util r_search
+BINDEPS+=r_core r_hash r_debug r_fs r_config r_bp r_bp r_crypto
```

Comment:
You also have to add the required dependencies in the `binr/ragg2/meson.build` file.
```
executable('ragg2', 'ragg2.c',
  include_directories: [platform_inc],
  link_with: [r_util, r_egg, r_bin],
  install: true
)
```
Obviously add it to the `link_with` line.

Reply:
Thanks a lot ! I pushed the modifications.